### PR TITLE
Vitalics common components

### PR DESCRIPTION
**Pull request by vitalics**
added missing typings

### CODE REVIEW

**Review by DrJekill352 — APPROVED** [review]

File: festival/frontend/src/commonApp/button/button.component.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<input type="text"> 123
```
Comment (ghost) [inline festival/frontend/src/commonApp/button/button.component.html]: sweet name for commit)

**Review by mkinitcpio — APPROVED** [review]

File: festival/frontend/src/commonApp/common.module.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+import { CommonModule } from '@angular/common';
+import { FormsModule } from '@angular/forms';
+import { MaterialModule } from '@angular/material';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+
+
+import {
+    InputComponent,
+    SpinnerComponent,
+<<<<<<< HEAD
```
Comment (zzz3bra) [inline festival/frontend/src/commonApp/common.module.ts]: resolve merge conflict maybe?

**Review by zzz3bra — CHANGES_REQUESTED** [review]
Resolve merge conflict in common.module.ts

**Review by zzz3bra — APPROVED** [review]
Will we use branching or now it is a deprecated technique?

**Pull request merged**